client/my-sites/plans/jetpack-plans/product-lightbox/index.tsx

```diff
@@ -25,7 +27,6 @@ import { Icons } from './icons/icons';
 import { Tags } from './icons/tags';
 import PaymentPlan from './payment-plan';
 import ProductDetails from './product-details';
-import ProductWithVariants from './product-with-variants';
```

Comment:
This is the custom component we were using to display the jetpack social products, we are getting rid of it now and use the same component that the backup product uses.

Reply:
Ah, that's another reason to use the pattern we already of have